go.mod

```diff
@@ -60,3 +60,5 @@ replace github.com/ElrondNetwork/arwen-wasm-vm/v1_2 v1.2.30 => github.com/Elrond
 replace github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.30 => github.com/ElrondNetwork/arwen-wasm-vm v1.3.30
 
 replace github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.8 => github.com/ElrondNetwork/arwen-wasm-vm v1.4.8
+
+replace github.com/ElrondNetwork/covalent-indexer-go => /home/elrond/go/src/github.com/ElrondNetwork/covalent-indexer-go
```

Comment:
proper release + remove this replace + go mod tidy

Reply:
Added release + removed it !